Podfile

```diff
@@ -79,7 +79,7 @@ end
 
 def shared_test_pods
   pod 'OHHTTPStubs/Swift', '~> 9.1.0'
-  pod 'OCMock', '~> 3.4.3'
+  pod 'OCMock', '~> 3.4'
```

Comment:
I didn't change this to `~> 3.9` because there's nothing there that we need. This way, in the rare occurrence one of our dependency requires a version of OCMock earlier than the lates, we'll be able to use it.